server/sonar-web/src/main/js/app/components/nav/app.js

```diff
@@ -24,13 +24,13 @@ import ReactDOM from 'react-dom';
 import GlobalNav from './global/global-nav';
 import ComponentNav from './component/component-nav';
 import SettingsNav from './settings/settings-nav';
-import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../api/nav';
+import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../../api/nav';
 
 export default class App {
   start () {
     const options = window.sonarqube;
 
-    require('../../components/workspace/main');
+    require('../../../components/workspace/main');
 
     return new Promise((resolve) => {
```

Comment:
@trajano hi, this is not the best place to ask such questions. It's better to ask in [SonarQube Google Groups](https://groups.google.com/forum/#!forum/sonarqube), the chance to get a good answer is much bigger.